index.js

```diff
@@ -0,0 +1,13 @@
+var express = require( "express" );
+var app, server, port;
```

Comment:
So we are un-opinionated about it or we use different var for each strictly now?